### PR TITLE
Remove extraneous m. in front of room events

### PR DESCRIPTION
**Pull request by rohan-wire**
globally replace `m.room.` with `room.` in event names.

### CODE REVIEW

File: draft-ralston-mimi-protocol.md
Comment (turt2live) [inline draft-ralston-mimi-protocol.md:1]: The prefix exists to ensure namespacing is possible, as folks may want to add their own event types to the protocol. For example, to accomplish DoubleRatchet use cases. 

It was my intention to also prefix the DS events with `m.` for consistency and building the official namespace. The namespace needs to be codified in the IANA registry still, but that was largely assumed to be a later problem.

`m.` is shorthand for `mimi.`, which we can expand if needed.

**Review by turt2live — CHANGES_REQUESTED** [review]
see threaded comment

**Comment by bifurcation** [comment]
Yeah, I'm inclined to leave this for the WG to comment on.  The traditional IETF approach to something like this would be to have an IANA registry where certain values are registered (those that would be `m.` here) and certain values would be vendor space (I believe the Matrix pattern for these is `com.example.foo`).  If we were talking about numerical code points, the analogy would be exact, since if you know the range of numbers reserved for vendor usage (≈ non-`m.` here), you can look at a value and tell if it's registered or vendor.

For text-like registries, things are a little less clear.  Traditionally, these registries don't have an explicit vendor reservation, since space is not constrained.  (See, e.g., the [ACME directory metadata fields registry](https://www.iana.org/assignments/acme/acme.xhtml#acme-directory-metadata-fields).)  If you want a value for some proprietary thing, you just choose one and use it, and if some official thing later collides with it, oh well.  And in cases where there were explicit affordances for experimental (e.g., `X-` and `P-` in HTTP/SIP headers), those affordances [are now deprecated](https://datatracker.ietf.org/doc/html/rfc6648).

So it's not immediately clear to me what the right answer is here, and how to do extensibility seems like a good question for the WG to weigh in on.

File: draft-ralston-mimi-protocol.md
Comment (rohan-wire) [reply draft-ralston-mimi-protocol.md:1]: I propose we use `private.` and a reverse domain name label for namespacing of private events. 
MIMI  events then can use room for room events, ds for ds events, etc.

Comment (rohan-wire) [reply draft-ralston-mimi-protocol.md:1]: yes

Comment (rohan-wire) [reply draft-ralston-mimi-protocol.md:1]: and if we use an enum managed by IANA then we can use `room` and `dr` but the `dr` constant might be initially in the private range.

Comment (rohan-wire) [reply draft-ralston-mimi-protocol.md:1]: If we use an enum instead for the IANA registry then we get the best of both worlds

**Comment by rohan-wire** [comment]
Closing as this is superfluous in PR #50

**Pull request closed without merging**